lib/contentfulPerson.js

```diff
@@ -23,11 +23,11 @@ const ContentfulPerson = {
 
     const response = await ContentfulApi.callContentful(query, variables);
 
-    const personCollection = response.data.personCollection.items
+    const personCollection = response.data?.personCollection.items
       ? response.data.personCollection.items
       : [];
 
-    return personCollection[0];
+    return personCollection.length > 0 ? personCollection[0] : null;
```

Comment:
Here even if `personCollection` is not present an empty Array would have caused a failure.